pony/migrate/writer.py

```diff
@@ -248,7 +248,7 @@ def ename(attr):
                 attr = None
                 if entity:
                     attr = entity._adict_.get(prev_attr.reverse.name)
-                    if attr.entity is not entity: assert False, 'Not implemented'
+                    if attr and attr.entity is not entity: assert False, 'Not implemented'
```

Comment:
assert not attr or attr.entity is entity, 'Not implemented'

Reply:
I guess, it is not necessary to change code style only here